### PR TITLE
Sighting model

### PR DESCRIPTION
**Pull request by thaisamarinho**
@blb451  @shofaizi 

please review the rest of the code. I forgot to commit the changes on the views and seeds in the last one. sorry about that.

### CODE REVIEW

**Review by shofaizi — COMMENTED** [review]
👍

**Comment by thaisamarinho** [comment]
@whilestevego 

Steve, more code review for you, still from the sightings model , i forgot to add and commit the views and seeds on the last pull. here they go. Thanks.

File: app/views/sightings/_form.html.erb [hunk 1]
```diff
@@ -11,23 +11,28 @@
         <div class="pet-other" style="display: none;">
           <%=f.input :pet_type, placeholder: 'Please type the specie of your pet', label: 'Species', input_html: { id: 'dingdong' } %>
         </div>
+         <%= f.input :color%>
```
Comment (whilestevego) [inline app/views/sightings/_form.html.erb]: Add space between `:color` and `%>`

File: app/views/sightings/_form.html.erb [hunk 2]
```diff
@@ -11,23 +11,28 @@
...
+         <%= f.input :size, collection: @size%>
```
Comment (whilestevego) [inline app/views/sightings/_form.html.erb]: Space

File: app/views/sightings/show.html.erb [hunk 3]
```diff
@@ -12,7 +12,7 @@
 <%= @sighting.note %>
 </div>
 <div>
-Last Seen: <%= @sighting.date_time %>
+Last Seen on: <%= @sighting.last_seen_date %> at <%@sighting.last_seen_time%>.
```
Comment (whilestevego) [inline app/views/sightings/show.html.erb]: Spaces at `<%` and `%>`

**Review by whilestevego — CHANGES_REQUESTED** [review]
Few minor changes. Looks good aside from that

**Review by whilestevego — APPROVED** [review]

**Pull request merged**